src/Solnet.Rpc/SolanaRpcBatchWithAsyncs.cs

```diff
@@ -0,0 +1,145 @@
+﻿using Solnet.Rpc.Core.Http;
+using Solnet.Rpc.Messages;
+using Solnet.Rpc.Models;
+using Solnet.Rpc.Types;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Solnet.Rpc
+{
+    public class SolanaRpcBatchWithAsyncs
```

Comment:
Missing comments! Won't nitpick about private and internals, but public stuff must be properly commented for IDE tooltips, etc.

Reply:
Methods bellow should be easy, just copy from IRpcClient

Reply:
On a second note, this is the class we're removing right? If so, then ignore these comments, but do apply them on the callback version!

Reply:
Class removed.